### ISSUE TITLE
S3 node (non-aws) needs a ignore ssl issues switch

### ISSUE THREAD

**Issue by accforgithubtest** [n8n-io/n8n#7239]
**Describe the bug**
S3 node (non-aws) needs a ignore ssl issues switch (similar to the http / other nodes)

**To Reproduce**
1. Self hosted minio:latest
2. n8n 1.5.1 version 
3. Caddy latest with self signed certs 
4. All set up via docker compose

Create a simple workflow to upload a test file to a test bucket in minio. 
You will get a error 
```
ERROR: unable to verify the first certificate
```

**Expected behavior**
Will be nice to have a a ignore ssl issues switch (similar to the http / other nodes)

**Environment (please complete the following information):**
1. Self hosted minio:latest
2. n8n 1.5.1 version 
3. Caddy latest with self signed certs 
4. All set up via docker compose

**Additional context**

**Comment by Joffcom** [comment]
Hey @accforgithubtest,

This sounds more like a feature request than a bug which would be better suited to our support forum where we collect those, I have created an internal ticket for this anyway which we are tracking as `NODE-801`.

Out of interest have you thought about using something like Lets Encrypt so that you don't need to use self signed certificates on your local network anymore? I ended up configuring a wildcard certificate for my home domain which I use for everything and it has made life a bit easier.

**Comment by accforgithubtest** [comment]
Thanks @Joffcom !

Agree this is not a bug, more of an enhancement / feature.

On a related, I was able to work around this error with NODE_TLS_REJECT_UNAUTHORIZED=0 as a environment var.
However below doesn't work
```
environment:
    NODE_EXTRA_CA_CERTS=/xcerts/local.host+1.pem   #/xcerts/local.host+1.crt
volumes:
   ${pwd}/xcerts:/xcerts/:ro
```

I would prefer to add my certs specifically using NODE_EXTRA_CA_CERTS, instead of using NODE_TLS_REJECT_UNAUTHORIZED which might not be a good idea.

Any idea why NODE_EXTRA_CA_CERTS doesn't work ? Or am I missing something with my configuration here ? 

Thanks  for the automated certs suggestion, thats one of the reasons I set up caddy in from of n8n to automate this. Getting a domain and automating the certs in on my todo list for the future but for now - I am in the early stages of figuring out what will be in my homelab and how I will set them up to talk to each other.
